app/components/Sidebar/components/DocumentLink.js

```diff
@@ -128,7 +128,14 @@ function DocumentLink(
   // Draggable
   const [{ isDragging }, drag] = useDrag({
     type: "document",
-    item: () => ({ ...node, depth, active: isActiveDocument }),
+    item: () => ({
+      ...node,
+      depth,
+      active: isActiveDocument,
+      permission: collection?.permission,
+      collectionId: collection?.id,
+      collectionName: collection?.name,
```

Comment:
We should store only the `id` here, if the name and other details are needed in the modal then they can be loaded from the collections store?